packages/storybook/jest.config.js

```diff
@@ -13,10 +13,10 @@ module.exports = {
   ],
   coverageThreshold: {
     global: {
-      branches: 60,
-      functions: 60,
-      lines: 60,
-      statements: 60
+      branches: 50,
```

Comment:
This makes me way more interested in setting up Jest projects to allow customizing coverage thresholds on a package-by-package basis. 👌 for now.